source/common/tracing/zipkin/span_context.cc

```diff
@@ -0,0 +1,103 @@
+#include "common/tracing/zipkin/zipkin_core_constants.h"
+#include "common/tracing/zipkin/span_context.h"
+
+namespace Zipkin {
+
+const std::string SpanContext::FIELD_SEPARATOR_ = ";";
+
+SpanContext::SpanContext(const Span& span) {
+  trace_id_ = span.traceId();
+  id_ = span.id();
+  parent_id_ = span.parentId();
+
+  for (const Annotation& annotation : span.annotations()) {
+    if (annotation.value() == ZipkinCoreConstants::CLIENT_RECV) {
+      annotation_values_.cr_ = true;
+    }
+    if (annotation.value() == ZipkinCoreConstants::CLIENT_SEND) {
+      annotation_values_.cs_ = true;
+    }
+    if (annotation.value() == ZipkinCoreConstants::SERVER_RECV) {
+      annotation_values_.sr_ = true;
+    }
+    if (annotation.value() == ZipkinCoreConstants::SERVER_SEND) {
+      annotation_values_.ss_ = true;
+    }
+  }
+
+  is_populated_ = true;
+}
+
+const std::string SpanContext::serializeToString() {
+  if (!is_populated_) {
+    return "0000000000000000;0000000000000000;0000000000000000";
```

Comment:
can you create static const and name it "UNINITIALIZED_CONTEXT" or something along this line